src/main/java/micdoodle8/mods/galacticraft/core/recipe/RecipeManagerGC.java

```diff
@@ -61,10 +64,21 @@ public static void loadRecipes()
     @SuppressWarnings("unchecked")
     private static void addUniversalRecipes()
     {
-    	Object meteoricIronIngot = ConfigManagerCore.recipesRequireGCAdvancedMetals ? GCItems.meteoricIronIngot : "ingotMeteoricIron";
+
+        if (Loader.isModLoaded("miscutils"))
```

Comment:
Hard no aslong as these are the only higher fuel tier options.

Reply:
got any sugestions to add to the list of fuels

Reply:
Cetane-Boosted Diesel, HOG, maybe a Plasma, EnderIO "Rocket Fuel", etc.

Reply:
dont realy see the point of the eio rocket fuel sins the recipe is removed